src/components/sidemenu/SideMenu.js

```diff
@@ -1,24 +1,31 @@
-import React, { Component } from 'react';
+	const SideMenu = ({ subMenu }) => {
+	if (!subMenu || subMenu.length === 0) {
+		return;
+	}
 
-export interface SideMenuProps {
-    subMenu: array;
-}
+	return (
+		//TO DO select the active sub menu item probably based off the URL passed to the API
```

Comment:
@PorismDominicSkinner and I have been had some discussions around this today and confirmed some assumptions with Ananda - we will likely only have Sub Menus on "Content-Pages" (article style - e.g. case studies, "about" page), and they will likely just be internal-hyperlink style headings to "scroll the user down" to a particular (sub?) heading in the content

Reply:
Removed the side menu